server/src/main/java/com/cloud/configuration/ConfigurationManagerImpl.java

```diff
@@ -453,6 +457,12 @@
             "Max length of vm userdata after base64 decoding. Default is 32768 and maximum is 1048576", true);
     public static final ConfigKey<Boolean> MIGRATE_VM_ACROSS_CLUSTERS = new ConfigKey<Boolean>(Boolean.class, "migrate.vm.across.clusters", "Advanced", "false",
             "Indicates whether the VM can be migrated to different cluster if no host is found in same cluster",true, ConfigKey.Scope.Zone, null);
+    protected final ConfigKey<Integer> KvmVmMigrateSpeed = new ConfigKey<>("Advanced", Integer.class, KVM_VM_MIGRATE_SPEED, "-1",
```

Comment:
Would it make sense to reduce the scope of these setting to Cluster level?

Reply:
changed to cluster scope